migrations/20221003090000-create-thumbnails-table.js

```diff
@@ -0,0 +1,72 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable(
+      'thumbnails',
+      {
+        id: {
+          type: Sequelize.INTEGER,
+          primaryKey: true,
+          autoIncrement: true,
+          allowNull: false,
+        },
+        file_id: {
+          type: Sequelize.INTEGER,
+          allowNull: false,
+          unique: 'unique_thumbnail',
+        },
+        type: {
+          type: Sequelize.STRING,
+          allowNull: false,
+          unique: 'unique_thumbnail',
+        },
+        size: {
+          type: Sequelize.BIGINT.UNSIGNED,
+        },
+        bucket_id: {
+          type: Sequelize.STRING(24),
+        },
+        bucket_file: {
+          type: Sequelize.STRING(24),
+        },
+        encrypt_version: {
+          type: Sequelize.STRING(20),
+        },
+        created_at: {
+          type: Sequelize.DATE,
+          defaultValue: Sequelize.fn('now'),
+        },
+        updated_at: {
+          type: Sequelize.DATE,
+          defaultValue: Sequelize.fn('now'),
+        },
+      },
+      {
+        uniqueKeys: {
+          unique_thumbnail: {
+            fields: ['file_id', 'type'],
+          },
+        },
+      },
+    );
+
+    await queryInterface.addConstraint('thumbnails', {
+      type: 'FOREIGN KEY',
+      fields: ['file_id'],
+      name: 'thumbnails_file_id_fkey',
+      references: {
+        table: 'files',
+        field: 'id',
+      },
+      onDelete: 'CASCADE',
+      onUpdate: 'CASCADE',
+    });
+  },
+
+  down: async (queryInterface) => {
+    await queryInterface.dropTable('thumbnails');
+    await queryInterface.removeConstraint(
```

Comment:
Remove the constraint before dropping the table first, otherwise I believe it will throw an error to remove a constraint of a non existing table